scripts/jinx.lic

```diff
@@ -372,6 +372,18 @@ module Jinx
   end
 end
 
+module Jinx
+  module Lookup
+    class << self
+      def find_asset_in(repo, name)
+        repo[:available].find do |asset|
+          URI.decode(File.basename(asset[:file])).eql?(name)
```

Comment:
`URI.decode` seems to be depreciated, at least in v2.7.2 of Ruby. I was able to change this to `URI.decode_www_form_component(File.basename(asset[:file])).eql?(name)` without it giving an undefined method.

Reply:
Updated this to use CGI.unescape which is available in Ruby 2.1 and 2.7 and doesn't have the same issue of trying to do too much in one method that made URI.unescape obsolete.